example/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java'
+apply plugin: "net.ltgt.apt"
+
+dependencies {
+    compile 'com.google.code.gson:gson:2.6.2'
+    compile 'com.google.auto.value:auto-value:1.2'
```

Comment:
When I use `apt 'com.google.auto.value:auto-value:1.2'`, the compile will fail, when I change to `compileOnly 'com.google.auto.value:auto-value:1.2'`, it builds, so should I use `compileOnly`?